client/src/containers/home/MapFragment.js

```diff
@@ -1,28 +1,81 @@
 import React from 'react';
 import { Tabs } from 'antd';
-import HereMap from '../../components/HereMap';
+import HereMapInteractive from '../../components/HereMapInteractive';
 
 const { TabPane } = Tabs;
 
 const dummyPoints = [
-  [44.4368, 26.1125],
-  [44.4398, 26.1195],
+  {
+    general_id: 75,
+    risk_category: 'U1',
+    lat: 44.34556,
+    lng: 26.206,
+  },
+  {
+    general_id: 74,
+    risk_category: 'U3',
+    lat: 44.43556,
+    lng: 26.106984,
+  },
+  {
+    general_id: 73,
+    risk_category: 'N/A',
+    lat: 44.39556,
+    lng: 26.156,
+  },
+  {
+    general_id: 72,
+    risk_category: 'RS I',
+    lat: 44.42556,
+    lng: 26.176,
+  },
+  {
+    general_id: 71,
+    risk_category: 'RS II',
+    lat: 44.40553,
+    lng: 26.186,
+  },
+  {
+    general_id: 70,
+    risk_category: 'RS III',
+    lat: 44.41339,
+    lng: 26.146,
+  },
```

Comment:
Could we connect this to [the mock JSON server](https://github.com/code4romania/seismic-risc/tree/master/client#use-mock-data-api) or would it be too big of an effort for this?
This should be solved by #92, but until then could we make use of this server so we wouldn't have static data? 🤔